src/content/developers/docs/data-availability.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Data availability 
```

Comment:
Meta: where do you see this living in the docs? Probably the advanced section? Inside or after directly after scaling?

Reply:
The advance section would be ideal. I also suggest place it after the page on scaling.

Reply:
Advanced section would make sense. Let's add it after scaling--I think DA is more than just scaling and needs its own page.